cypress/steps/channelsSteps.js

```diff
@@ -1,26 +1,28 @@
 import { ADD_CHANNEL_FORM_SELECTORS } from "../elements/channels/add-channel-form-selectors";
 import { CHANNELS_SELECTORS } from "../elements/channels/channels-selectors";
 
-class ChannelsSteps {
-  createChannelByView(name, currency, otherCurrency, slug = name) {
-    cy.get(CHANNELS_SELECTORS.createChannelButton)
-      .click()
-      .get(ADD_CHANNEL_FORM_SELECTORS.channelName)
-      .type(name)
-      .get(ADD_CHANNEL_FORM_SELECTORS.slug)
-      .type(slug)
-      .get(ADD_CHANNEL_FORM_SELECTORS.currency)
+export function createChannelByView(
+  name,
+  currency,
+  otherCurrency,
+  slug = name
+) {
+  cy.get(CHANNELS_SELECTORS.createChannelButton)
+    .click()
+    .get(ADD_CHANNEL_FORM_SELECTORS.channelName)
+    .type(name)
+    .get(ADD_CHANNEL_FORM_SELECTORS.slug)
+    .type(slug)
+    .get(ADD_CHANNEL_FORM_SELECTORS.currency)
+    .click();
+  if (!otherCurrency) {
```

Comment:
Is other currency argumeent needed? This function could check if add currency option is available and click it if visible

Reply:
It's not. I removed otherCurrency